phoenix-core/src/main/java/org/apache/phoenix/hbase/index/IndexRegionObserver.java

```diff
@@ -131,8 +124,8 @@ public long getCount() {
           return count;
       }
 
-      public long getLatestTimestamp() {
-          return latestTimestamp;
+      public boolean isConcurrent() {
+          return concurrent;
```

Comment:
Nit: We don't need a boolean. We can just return count > 1;

Reply:
I cannot do that because the count is a reference count. I would not know if the row has gone through concurrent updates when the reference count drops to 1.